spyder/plugins/debugger/widgets/main_widget.py

```diff
@@ -377,6 +449,18 @@ def update_actions(self):
             action = self.get_action(action_name)
             action.setEnabled(pdb_prompt)
 
+        rows = self.breakpoints_table.selectionModel().selectedRows()
+        c_row = rows[0] if rows else None
+
+        enabled = (bool(self.breakpoints_table.model.breakpoints)
+                   and c_row is not None)
```

Comment:
```suggestion
        initial_row = rows[0] if rows else None

        enabled = (
            bool(self.breakpoints_table.model.breakpoints)
            and initial_row is not None
        )
```

I think `c_row` stands for `current_row`, but it's always referencing the initial row. So, I renamed like that.